webcompat/static/js/lib/bugform.js

```diff
@@ -176,25 +176,23 @@ function BugForm() {
       $("[value=" + problemType[1] + "]").click();
     }
 
-    // If we have one or more details params,
+    // If we have a details params, JSON decode it and
```

Comment:
Yeah, it's correct English because it's a singular param now (with a confusingly plural name). Less confusing would be something like `// If we have details, JSON decode it...`. I'll update.